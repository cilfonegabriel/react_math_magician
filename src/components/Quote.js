const Quote = () => (
  <div className="home-section">
    <h4 className="sentence">
      &quot;Since you are not experienced in things of the world
      <br />
      all things that have some difficulty seem impossible to you.
      <br />
      Trust in time, which usually gives sweet solutions to many bitter difficulties&quot;
    </h4>
    <p className="author"> - DON QUIJOTE </p>
  </div>
);

export default Quote;
