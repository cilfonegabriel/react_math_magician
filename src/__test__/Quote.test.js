import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

test('Home snapshot', () => {
  const tree = renderer.create(<Quote />);
  expect(tree).toMatchSnapshot();
});
