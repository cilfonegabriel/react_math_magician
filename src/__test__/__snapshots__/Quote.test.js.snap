// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot 1`] = `
<div
  className="home-section"
>
  <h4
    className="sentence"
  >
    "Since you are not experienced in things of the world
    <br />
    all things that have some difficulty seem impossible to you.
    <br />
    Trust in time, which usually gives sweet solutions to many bitter difficulties"
  </h4>
  <p
    className="author"
  >
     - DON QUIJOTE 
  </p>
</div>
`;
